src/RenderingSystem.cpp

```diff
@@ -71,34 +77,23 @@ void RenderingSystem::handle_key_press(const Event& e) {
 void RenderingSystem::render() {
     start_render();
 
-    for (auto& object : example_objects_) {
-        object.render_views(cameras_, 4, example_shader_.program_id_);
-    }
-
-    end_render();
-}
-
-bool RenderingSystem::init_window() {
-    SDL_Init(SDL_INIT_EVERYTHING);
-
-    const int sdl_flags = SDL_WINDOW_SHOWN | SDL_WINDOW_OPENGL;
 
-    const int screen_width = 640;
-    const int screen_height = 480;
+    for (size_t i = 0; i < cameras_.size(); i++) {
+        GLint x = 320 * (i % 2);
+        GLint y = 240 - 240 * (i / 2);
 
-    window_ = SDL_CreateWindow("WienerTakesAll",
-                               SDL_WINDOWPOS_UNDEFINED,
-                               SDL_WINDOWPOS_UNDEFINED,
-                               screen_width,
-                               screen_height,
-                               sdl_flags);
+        glViewport(x, y, 320, 240);
 
-    if (window_ == NULL) {
-        std::cout << "Window could not be created! SDL Error: " << SDL_GetError() << std::endl;
-        return false;
+        for (auto& object : example_objects_) {
+            object.render(cameras_[i]);
+        }
     }
 
 
+    end_render();
```

Comment:
I know it's not part of this PR, but I think these names (start_render and end_render) are misleading, `start_render` to me implies something is happening in the background (like a different thread), but that's not the case.

I think `init_render` and `finish_render` would be better.